### ISSUE TITLE
[BUG] Bundles must have unique filePaths with multiple HTML assets

### ISSUE THREAD

**Issue by jcsirot** [101arrowz/pwa-manifest#22]
## Describe the bug

With that structure
```
my-project/
├── pages/
│   └── about.html
├── index.html
├── .parcelrc
└── package.json
```
When the project contains several html assets the `parcel index.html` command fails with this error 
```
$ parcel index.html
ℹ️ Server running at http://localhost:1234
🚨 Build failed.
Error: Bundles must have unique filePaths
AssertionError [ERR_ASSERTION]: Bundles must have unique filePaths
    at BundlerRunner.nameBundles (/Users/jcsirot/git/my-project/node_modules/@parcel/core/lib/BundlerRunner.js:333:23)
```

## Expected behavior
The project should be built with all pages including the same web manifest.

## Actual behavior
The build fails

## To reproduce
Create a simple PWA project with 2 HTML assets (index.html including a link to about.html in my example). Run `parcel index.html`

**Comment by 101arrowz** [comment]
I can reproduce this, but fixing may take a bit. Thanks for the bug report.

**Comment by 101arrowz** [comment]
Will be fixed along with #40 in the next release, thanks for the report!
